src/operator/lrn.cc

```diff
@@ -38,9 +38,26 @@ Operator* LocalResponseNormProp::CreateOperatorEx(Context ctx, std::vector<TShap
 DMLC_REGISTER_PARAMETER(LRNParam);
 
 MXNET_REGISTER_OP_PROPERTY(LRN, LocalResponseNormProp)
-.add_argument("data", "NDArray-or-Symbol", "Input data to the ConvolutionOp.")
+.add_argument("data", "NDArray-or-Symbol", "Input data.")
 .add_arguments(LRNParam::__FIELDS__())
-.describe("Apply convolution to input then add a bias.");
+.describe(R"code(Applies Local Response Normalization to the input.
+
+Normalization helps in generalization and used to prevent neurons from 
```

Comment:
Maybe be more descriptive here. Not all neurons can saturate. Also there is a grammatical bug on "used to" (suggests in the past, not utility). 

Perhaps: "Normalization helps to prevent squashing neurons (like tanh) from saturating. It can also help for transfer learning, as the inputs for each task will exhibit the same dynamic range."